kb/sd_artificial_neural_networks/concepts/cd_architectures_added_to_describe_methods_for_solving_problems/concept_echo_state_ann.scs

```diff
@@ -0,0 +1,6 @@
+concept_echo_state_ann
```

Comment:
Add full concept formalization. Mode all ann classes to sd_artificial_neural_networks